libsupport/include/katana/EntityTypeManager.h

```diff
@@ -420,10 +469,18 @@ class KATANA_EXPORT EntityTypeManager {
     static_assert(kUnknownEntityType == 0);
     // add kUnknownEntityType: do not treat it as an atomic type;
     // treat it as an entity type that does not have any atomic subtypes
-    auto id = AddNonAtomicEntityType(SetOfEntityTypeIDs());
+    SetOfEntityTypeIDs empty_type_id_set;
+    empty_type_id_set.resize(kDefaultSetOfEntityTypeIDsSize);
+    auto id = AddNonAtomicEntityType(empty_type_id_set);
     KATANA_LOG_ASSERT(id.value() == kUnknownEntityType);
   }
 
+  /// The current size of the SetEntityTypeIDs bitsets
+  size_t SetOfEntityTypeIDsSize_ = kDefaultSetOfEntityTypeIDsSize;
```

Comment:
constants should start `k`. Why isn't this a static constexpr?

Reply:
its not constant, otherwise our bitsets wouldn't be dynamic!

Reply:
Then CamelCase is confusing. Use snake_case_. But just `size_` is fine right?